### DISCUSSION TITLE
Hello, I have a problem with "git clone"

### DISCUSSION THREAD

**Discussion by H055am** [community/community#53493] [Repositories] [answered]
### Select Topic Area

Question

### Body

I need to know what is the cause of the error.
 Note: I deleted the "repo" and create another one, and I had the same problem.
![Screenshot (726)](https://user-images.githubusercontent.com/125449695/233793164-bbe54945-5414-45fe-b926-986397f09795.png)

**Comment by airtower-luna** [comment]
That's a DNS problem. The screenshot looks like you have a Ubuntu VM in VMware on Windows, if your Windows system can connect to GitHub most likely there's some error in the network settings for your VM (could be in VMware or the VM itself).

**Reply by H055am** [reply]
I don't think so because I've been working on it for months.
I created a lot of "repo" and I was cloning and editing and then uploading files easily.

**Reply by airtower-luna** [reply, marked as answer]
Something must've changed. What's obvious is this:

* You VM has a DNS problem.
* The network applet in the panel at the top of the VM screen is showing an error indicator, too.

You'll have to figure out why, no-one here can remotely look at your VM network setup.

**Reply by H055am** [reply]
Well you were right
It was a network problem in the VM
I finished it and the repo was copied successfully.
Thank you

**Reply by airtower-luna** [reply]
I'm glad it's working now! Please don't forget to mark the answer. :slightly_smiling_face:
